tests/TestCase.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Letsgoi\HealthCheck\Tests;
+
+use Letsgoi\HealthCheck\HealthCheckServiceProvider;
+use Orchestra\Testbench\TestCase as BaseTestClass;
+
+abstract class TestCase extends BaseTestClass
+{
+    protected function getPackageProviders($app): array
```

Comment:
`$app` type?

Reply:
i can't put the type because is an inherited method from BaseTestClass.

However for now, we don't need inject the providers, so I will remove it.